### PR TITLE
Code update

### PR DESCRIPTION
**Pull request by Quatrecentquatre-404**
I separated the code in a ``src`` folder to make it more readable and being able to use it as a module in platform such as  NPM, I replaced ``node-fetch`` by ``axios`` because it's easier to get JSON response data rather than using two ``await`` blocks.

### CODE REVIEW

**Comment by overlisted** [comment]
Oh, I didn't know that anyone uses this! Give me 1 minute to review this PR

File: README.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-- Install NPM if you don't have it;
-- Run command "npm i" in the repository's directory;
-- Paste [your Discord token](https://youtu.be/xuB1WQVM3R8) to (new) file token.txt;
-- Copy the channel's [ID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID)
-- Run the script with command `node index.js <channel ID you've copied>`;
-- A complete list of all messages in that channel is now in the file messages.txt;
+# Usage
+-   Clone this repository :
+```bash
+git clone https://github.com/Quatrecentquatre-404/discord-messages-fetch # Clone the repository
```
Comment (overlisted) [inline README.md:7]: this links to your fork
```suggestion
git clone https://github.com/overlisted/discord-messages-fetch # Clone the repository
```

File: package.json [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
-  "main": "index.js",
+  "version": "1.0.0",
+  "description": "Quickly fetch whole message history of a Discord channel",
+  "main": "test.js",
+  "scripts": {
+    "start": "echo \"Usage :\nnode test.js <channel_id>\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Quatrecentquatre-404/discord-messages-fetch.git"
```
Comment (overlisted) [inline package.json:11]:
```suggestion
    "url": "git+https://github.com/overlisted/discord-messages-fetch.git"
```

File: package.json [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Quatrecentquatre-404/discord-messages-fetch/issues"
+  },
+  "homepage": "https://github.com/Quatrecentquatre-404/discord-messages-fetch#readme",
```
Comment (overlisted) [inline package.json:19]: these fields are meant for libraries

**Review by overlisted — CHANGES_REQUESTED** [review]
This is also generally not my favorite code style, but I don't really care because it's just a little script :)

**Comment by Quatrecentquatre-404** [comment]
> This is also generally not my favorite code style, but I don't really care because it's just a little script :)

Yeah, it's kinda huge for this little boy, but the a dev comes here to check how it works, it's pretty simple right now because he knows the main thing will be in the ``src/`` folder. He doesn't have to crawl on the ``test.js`` file or else to see the engine of this project.

**Review by overlisted — APPROVED** [review]

**Pull request merged**